packages/sdk/package.json

```diff
@@ -30,6 +30,7 @@
     "bitcoinjs-lib": "6.1.3",
     "bitcoinjs-message": "2.2.0",
     "buffer-reverse": "^1.0.1",
+    "cbor-js": "^0.1.0",
```

Comment:
Can we replace it with this? https://www.npmjs.com/package/cbor

chor-js was last released 8 years ago.